frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -2,13 +2,16 @@ import React from 'react'
 import Explanation from './Explanation/Explanation'
 
 //Import components that you create here
-
+import Tools from './Tools/Tools'
+import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
If you check the index.js, the min file has already been imported there so there is no need to import here as well.